app/src/main/java/com/github/libretube/ui/dialogs/SubmitDeArrowDialog.kt

```diff
@@ -56,7 +57,17 @@ class SubmitDeArrowDialog: DialogFragment() {
             .setNegativeButton(R.string.cancel, null)
             .show()
             .apply {
-                getButton(DialogInterface.BUTTON_POSITIVE).setOnClickListener {
+                val positiveButton = getButton(DialogInterface.BUTTON_POSITIVE)
+                positiveButton.isEnabled = false
+
+                val onChecked = CompoundButton.OnCheckedChangeListener { _, _ ->
+                    positiveButton.isEnabled = binding.titleCheckbox.isChecked || binding.thumbnailTimeCheckbox.isChecked
+                }
+
+                binding.titleCheckbox.setOnCheckedChangeListener(onChecked)
+                binding.thumbnailTimeCheckbox.setOnCheckedChangeListener(onChecked)
```

Comment:
We already set onCheckedChangeListener's above, it's not possible to add multiple checked change listeners. Hence this results in none of the text input fields being usable anymore.